src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -300,85 +302,6 @@ public String toString() {
                 ')';
     }
 
-    @AutoService(LoadFlowParameters.ConfigLoader.class)
-    public static class OpenLoadFlowConfigLoader implements LoadFlowParameters.ConfigLoader<OpenLoadFlowParameters> {
```

Comment:
Here the class is removed and the loading is moved within `OpenLoadFlowProvider::loadSpecificParameters`, I'm ok to put this class in the provider but why not keep the class? Simply replacing 
`LoadFlowParameters.ConfigLoader<OpenLoadFlowParameters>` by `ExtensionConfigLoader<LoadFlowParameters, OpenLoadFlowParameters>`